### DISCUSSION TITLE
small lathe big 3-phase stepping motor

### DISCUSSION THREAD

**Discussion by Leygan1** [kachurovskiy/nanoels#111] [Q&A]
Hello to all!

Trying to make NanoEls H2 already made the board and now its time for stepper...

I have [VRDM 3910/50](https://datasheetspdf.com/pdf-file/680900/Schneider/VRDM3910/1) for motor its 3f I have and smaller secret brand 3 faze stepper but absolutely no info about it.

For Driver cheapest alliexpres tb6600 buth I want to use Berger [LAHR WDP5-228](https://www.se.com/ww/en/download/document/00441106140/) 

I am not weary smart in stepper stuff will this assembly will work? if yes I will try to figure out how to set it up.

Gona try this setup on Halifax (Atlas)

**Comment by kachurovskiy** [comment]
Hi @Leygan1, I've never used steppers and drivers like this so it's very hard to say if they will work together.

Driver seems to support the DIR/PUL 5V logic NanoEls H2 uses as per page 30 of [this doc](https://download.schneider-electric.com/files?p_enDocType=User+guide&p_File_Name=Wdp5228e.pdf&p_Doc_Ref=00441106140). Page 53 seems to explain which wires to connect Pulse and Direction to. That said the manual is mighty long and I didn't actually read it :)

Cheers!

**Reply by Leygan1** [reply]
Well yes, the manual is big mostly about serial connection. in Any case, it seems to me that I can't control the 3fhaze stepper with this unit I will have to buy something else :(
